src/app/loadout-builder/filter/EnergyOptions.m.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../variables';
+
+.energyOptions {
+  margin: 8px 0;
+}
+
+.settingGroup {
+  background-color: black;
+  padding: 8px;
+}
+
+.title {
+  margin-bottom: 4px;
+}
+
+.buttons {
+  composes: flexRow from '../../dim-ui/common.m.scss';
+}
+
+.button {
+  cursor: pointer;
+  flex-grow: 1;
+
+  &:hover,
+  &:active {
+    background-color: #e2af55 !important;
```

Comment:
I kinda picked a random colour to distinguish hover from selected. Selected is using the standard button hover/selected colour.